src/components/AppBar.jsx

```diff
@@ -101,42 +159,125 @@ export default function AppBar() {
-              <TextField
-                id="search-field"
-                placeholder="Search..."
-                variant="outlined"
-                size="small"
+        <Hidden smDown>
+          <div className={classes.navigation}>
+            <Container fixed className={classes.menu}>
+              <Typography className={classes.menuItems}>
+                <NavLink
+                  to="/dashboard"
+                  activeStyle={{ color: activeLinkColor }}
+                >
+                  Dashboard
+                </NavLink>
+                <NavLink to="/explore" activeStyle={{ color: activeLinkColor }}>
+                  Explore
+                </NavLink>
+                <NavLink to="/search" activeStyle={{ color: activeLinkColor }}>
+                  Search
+                </NavLink>
+              </Typography>
+              <div className={classes.search}>
+                <TextField
+                  id="search-field"
+                  placeholder="Search..."
+                  variant="outlined"
+                  size="small"
+                  InputProps={{
+                    classes: {
+                      input: classes.searchInput
+                    }
+                  }}
+                />
+              </div>
+            </Container>
+          </div>
+        </Hidden>
+        <div>
+          <Hidden smDown>
+            <IconButton>
+              <ChatIcon />
+            </IconButton>
+            <IconButton>
+              <NotificationsIcon />
+            </IconButton>
+          </Hidden>
+          <Hidden smDown>
+            <Button
+              className={classes.accountButton}
+              aria-controls="fade-menu"
+              aria-haspopup="true"
+              endIcon={open ? <ArrowDropUpIcon /> : <ArrowDropDownIcon />}
+              color="inherit"
+              variant="outlined"
+              onClick={handleClick}
+            >
+              Alexander Bergholm
+            </Button>
+          </Hidden>
+          <Hidden mdUp>
+            <IconButton>
+              <AccountCircleIcon
+                aria-controls="fade-menu"
+                aria-haspopup="true"
+                color="inherit"
+                onClick={handleClick}
+                fontSize="default"
               />
-            </div>
-          </Container>
-        </div>
-        <div className={classes.account}>
-          <IconButton>
-            <ChatIcon />
-          </IconButton>
-          <IconButton>
-            <NotificationsIcon />
-          </IconButton>
-          <IconButton>
-            <AccountCircleIcon />
-          </IconButton>
+            </IconButton>
+          </Hidden>
+
+          <StyledMenu
+            id="account-menu"
+            anchorEl={anchorEl}
+            open={open}
+            onClose={handleClose}
+            keepMounted
+          >
+            <MenuList className={classes.menuList}>
+              <Hidden mdUp>
```

Comment:
It seems that there is some weirdness going on when rendering `Hidden` as a child of `MenuList`. It looks like `MenuList` is passing a `tabIndex` prop to all its children, but `Hidden` doesn't like it and produces a warning in the console. I found this comment here that might be helpful: https://github.com/mui-org/material-ui/issues/12468#issuecomment-412238375

Reply:
Good point--I didn't notice this before. I've resolved it using the comment you linked. Thanks!